front/src/components/eventItem.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="listRow">
+    <div>
+      <span>{{event.title}}</span>
+    </div>
+    <div>
+      <span>{{event.date}}</span>
+    </div>
+    <div>
+      <span>{{event.place}}</span>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "eventItem",
+  props: ["event"]
+};
+</script>
+
+<style>
+.listRow {
+  display: flex;
+}
+.listRow * {
+  flex-grow: 1;
+  flex-basis: 25%;
+}
+span {
```

Comment:
Product owner told us to not use html elements as selectors
```suggestion
.eventProperty{
```